internal/pkg/plugin/trellogithub/trellogithub.go

```diff
@@ -257,33 +257,34 @@ type TrelloItemId struct {
 }
 
 // CreateTrelloItems create board/lists, and set secret by ids
+// TODO(daniel-hutao): rename the function name
 func (gi *TrelloGithub) CreateTrelloItems() (*TrelloItemId, error) {
 	c, _ := trello.NewClient()
 	board, err := c.CreateBoard("DevStream_Trello_Board")
 	if err != nil {
 		return nil, err
 	}
 
-	todo, err := c.CreateList(board, "TODO")
+	lists, err := board.GetLists()
 	if err != nil {
 		return nil, err
 	}
-
-	doing, err := c.CreateList(board, "DOING")
-	if err != nil {
-		return nil, err
+	if len(lists) != 3 {
+		log.Errorf("Unknown lists format: len==%d", len(lists))
+		return nil, fmt.Errorf("unknown lists format: len==%d", len(lists))
 	}
 
-	done, err := c.CreateList(board, "DONE")
-	if err != nil {
-		return nil, err
-	}
+	todo := lists[0].ID
```

Comment:
Please add a comment here for future reference. Document why we process it by order. In case Trello changes its API or order in the future, we can quickly locate the bug.